src/njsVariable.c

```diff
@@ -235,6 +239,8 @@ uint32_t njsVariable_getDataType(njsVariable *var)
             return NJS_DATATYPE_CLOB;
         case DPI_ORACLE_TYPE_BLOB:
             return NJS_DATATYPE_BLOB;
+        case DPI_ORACLE_TYPE_BOOLEAN:
+            return NJS_DATATYPE_BOOLEAN;
```

Comment:
This section is not needed as booleans cannot be fetched. They can only be used in PL/SQL.